leaflet/leaflet.d.ts

```diff
@@ -256,11 +259,43 @@ declare namespace L {
         // Extension methods
         onAdd(map: Map): this;
         onRemove(map: Map): this;
-        getEvents(): {[name: string]: (event: Event) => void};
+        getEvents(): { [name: string]: (event: Event) => void };
         getAttribution(): string;
         beforeAdd(map: Map): this;
     }
 
+    export interface LayerGroup extends Layer {
+        initialize(layers: Layer[]);
+        addLayer(layer: Layer): this;
+        removeLayer(layer: Layer): this;
+        hasLayer(layer: Layer): boolean;
+        clearLayers(): this;
+        invoke(methodName: string): this;
+        eachLayer(fn: (layer: Layer) => void, context?: Object): this;
+        getLayer(id: string): Layer;
+        getLayers(): Layer[];
+        setZIndex(index: number);
+        getLayerId(layer: Layer): string;
+    }
+
+    export interface FeatureGroup extends LayerGroup {
+        setStyle(options: PathOptions): this;
+        bringToFront(): this;
+        bringToBack(): this;
+        getBounds(): LatLngBounds;
+    }
+
+    export interface GeoJSONOptions {
+        pointToLayer: (feature: any, latLng: LatLng) => void;
+        style: (feature: any) => void;
+        onEachFeature: (feature: any, layer: Layer) => void;
+        filter: (feature: any) => boolean;
+        coordsToLatLng: (coords: number[]) => LatLng;
```

Comment:
The parameter's type can be more specific: `[number, number] | [number, number, number]`